inspire_query_parser/visitors/elastic_search_visitor.py

```diff
@@ -705,95 +703,100 @@ def visit_keyword(self, node):
         # If no keyword is found, return the original node value (case of an unknown keyword).
         return self.KEYWORD_TO_ES_FIELDNAME.get(node.value, node.value)
 
-    def visit_value(self, node, fieldnames=None):
-        if not fieldnames:
-            fieldnames = '_all'
-
-        if node.contains_wildcard:
-            if self.KEYWORD_TO_ES_FIELDNAME['date'] == fieldnames:
-                return self._generate_date_with_wildcard_query(node.value)
+    def visit_value_wildcard(self, node, fieldnames=None):
```

Comment:
I think this is a wrong name, as it may clash with a `ValueWildcard` in the AST due to https://github.com/inspirehep/inspire-query-parser/blob/fcf52de7912b59455b0107b1b61b5f2f2b2d74e4/inspire_query_parser/visitors/visitor_impl.py#L41-L45
Maybe something like
```suggestion
    def handle_value_wildcard(self, node, fieldnames=None):
```